tests/cover/test_database_usage.py

```diff
@@ -33,17 +33,17 @@ def has_a_non_zero_byte(x):
 
 def test_saves_incremental_steps_in_database():
     key = b"a database key"
-    database = SQLiteExampleDatabase(':memory:')
```

Comment:
👍 I'm always a fan of de-crufting things, so I did a `git grep` for this pattern and found a few more.  Can you check the remaining uses and remove them if (as it seems to me) the new in-memory DB would be simpler?

Reply:
I propose a separate PR in which I deprecate the SQLiteExampleDatabase entirely (it's pure legacy and there's no good reason to use it any more) which will result in sorting all of that out.

Reply:
Sounds good to me.